preupg/cli.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function, unicode_literals
-#import argparse
 import optparse
```

Comment:
In future, we would use argparse if it is available and fallback to optparse otherwise. I created an issue #223 for such change.